swagger_info.go

```diff
@@ -0,0 +1,60 @@
+package swag
+
+import (
+	"bytes"
+	"encoding/json"
+	"strings"
+	"text/template"
+)
+
+// DocumentationReader is an interface to inject Swagger info reader.
+type DocumentationReader interface {
+	Swagger
+	InstanceName() string
+}
+
```

Comment:
A good practice is to define the interface where you use it not where you implement it. Please remove this definition ...

Reply:
done